aws-cloudwatch-alarm/main.tf

```diff
@@ -1,5 +1,12 @@
+locals {
+  enable_alarms = var.md_metadata.observability.alarm_channels.aws != null
+}
 resource "aws_cloudwatch_metric_alarm" "alarm" {
+  count      = local.enable_alarms ? 1 : 0
   alarm_name = var.alarm_name
+
+  # We need to 'smuggle' our name_prefix for the package back to massdriver
+  # so we can show the alarms on the correct manifest in the UI
```

Comment:
@xpositivityx I dont think we'll need to do this anymore after your changes to add `massdriver_alarm` to our tf provider since you'll be able to bind a package to a cloud's alarm, right?

Not suggesting we pull it now, but if thats true, it'll be nice to have _one way_ to assoc a package to a cloud alarm.

Reply:
Yeah we can remove this but I want to get the solution in place. Should be BC so its all good.